frontend/stylesheets/main.less

```diff
@@ -324,8 +323,8 @@ button.narrow {
   top: 100px;
   width: 400px;
   height: 250px;
-  background: var(--ant-modal-content-bg);
-  border-radius: 4px;
+  background: var(--ant-color-bg-container);
```

Comment:
The ant modal background color vars are not available globally but only for `<Modal  />` components. :-/